docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,378 @@
+---
+---
+# Kubernetes Federated Services and Cross-Cluster Service Discovery
```

Comment:
Remove; the title of the document should be in /_data/guides.yml, where this page must be added in order to show up in the left-navigation.